Makefile

```diff
@@ -0,0 +1,34 @@
+PLUGINS = $(sort \
+		  $(subst /, -, \
+		  $(patsubst cmd/%/, %, \
+		  $(dir \
+		  $(shell find cmd/ -type f -name '*.go')))))
+
+DOCKERFILES = $(sort \
+			  $(subst /, -, \
+			  $(patsubst cmd/%/, %, \
+			  $(dir \
+			  $(shell find cmd/ -type f -name 'Dockerfile')))))
+
+build: $(patsubst %, build-%, $(PLUGINS))
+
+build-%:
+	cd cmd/$(subst -,/,$*) && go build
+
+test:
+	go test ./cmd/... ./pkg/...
+
+test-%:
+	go test ./$(subst -,/,$*)/...
+
+docker-build: $(patsubst %, docker-build-%, $(DOCKERFILES))
+
+docker-build-%:
+	sudo docker build -t quay.io/kubevirt/device-plugin-$*:latest cmd/$(subst -,/,$*)
```

Comment:
Why the sudo? That shouldn't be needed if you're in the `docker` group.

Reply:
Done